docs/theory-of-operation.md

```diff
@@ -72,28 +72,65 @@ chip. To enter Hidden menu type `shift + z` in Payload menu.
   q        - exit menu
 ```
 
-## coreboot features
+## Coreboot features
```

Comment:
@piotr-kleins coreboot always starting from the small letter - even at the beginning of the sentence.